lib/github/search.rb

```diff
@@ -2,9 +2,6 @@
 
 module GitHub
   class Search
-    REPOSITORY_REGEX = /[a-zA-Z0-9\._-]+/
-    USERNAME_REGEX = /[a-zA-Z0-9_-]+/
```

Comment:
I just moved these regexes from the `GitHub::Search`module to the `GitHub` module.